PWGMM/Mult/Tasks/dndeta-hi.cxx

```diff
@@ -361,8 +348,10 @@ struct MultiplicityCounter {
         if (btrigc[kSel8])
           btrigc[kSel8g0] = true;
       }
+      if (btrigc[kSel8])
+        registry.fill(HIST("reczINELg0Sel8"), z);
 
-      float cent = -1;
+      auto cent = -1.0;
```

Comment:
Floating point number is considered `double` by default, if you want to restrict it to float you need to add a suffix
```cpp
auto cent = -1.f;
```
Also in the instances below.